dev-libs/libratbag/metadata.xml

```diff
@@ -1,8 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+
```

Comment:
This empty line seems non-standard...
(The same in ` app-misc/piper/metadata.xml`.)

Reply:
Fixed both ebuilds.